### DISCUSSION TITLE
_finalizeDimensions in V5 should I use that in V6?

### DISCUSSION THREAD

**Discussion by StringKe** [fabricjs/fabric.js#9539] [Q&A] [answered]
I need the group to ignore the strokewdith of the element when calculating the boundingbox.

**Comment by ShaMan123** [comment, marked as answer]
I don't remember what `_finalizeDimensions` did
In v6 you call subclass a LayoutStrategy

```ts
class MyLayout extends fabric.FitContentLayout {
calcBoundingBox(
    objects: FabricObject[],
    context: StrictLayoutContext
  ): LayoutStrategyResult | undefined {
    // handle the measuring logic here
   return super.calcBoundingBox(objects, context)
  }
}

new fabric.Group(object, { ...options, layoutManager: new fabric.LayoutManager(new MyLayout()) })
```

**Reply by ShaMan123** [reply]
This is how an object is measured
If you want to hack it simply set strokeWidth to zero and restore it after measuring
https://github.com/fabricjs/fabric.js/blob/7154d143f0e34bf3e779b5b9592ca98e704ea2fd/src/LayoutManager/LayoutStrategies/utils.ts#L16-L53
